neqo-transport/src/connection/mod.rs

```diff
@@ -1532,7 +1532,7 @@ impl Connection {
     /// This takes two times: when the datagram was received, and the current time.
     fn input(&mut self, d: &Datagram, received: Instant, now: Instant) {
         // First determine the path.
-        let path = self.paths.find_path_with_rebinding(
+        let path = self.paths.find_path(
```

Comment:
For others, relevant discussion happening on https://github.com/mozilla/neqo/pull/2176#pullrequestreview-2378912679.